presto-main/src/main/java/com/facebook/presto/operator/aggregation/state/BooleanDistinctState.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.state;
+
+import com.facebook.presto.spi.function.AccumulatorState;
+
+public interface BooleanDistinctState
+        extends AccumulatorState
+{
+    int getInt();
```

Comment:
use `byte` since `StateCompiler` already supports it: 

https://github.com/prestodb/presto/blob/79f480bc807954c4486b670c42015fb9c396f010/presto-main/src/main/java/com/facebook/presto/operator/aggregation/state/StateCompiler.java#L107-L109

Reply:
thanks, updated